File: 1-easy/problems/1389-create-target-array-in-the-given-order/solution.ts
function createTargetArray(nums: any, index: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :type index: List[int]
  :rtype: List[int]
  """
  for (i in Array.from({length: (nums}, (_, i) => i).length)) {
      for (j in Array.from({length: i}, (_, i) => i)) {
          if (index[j] >= index[i]) {
              index[j] += 1
  result = [0]*((nums).length)
  for (i in Array.from({length: (nums}, (_, i) => i).length)) {
      result[index[i]] = nums[i]
  return result;
}

export { createTargetArray };

File: 1-easy/problems/1389-create-target-array-in-the-given-order/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def createTargetArray(self, nums, index):
        """
        :type nums: List[int]
        :type index: List[int]
        :rtype: List[int]
        """
        for i in range(len(nums)):
            for j in range(i):
                if index[j] >= index[i]:
                    index[j] += 1
        result = [0]*(len(nums))
        for i in range(len(nums)):
            result[index[i]] = nums[i]
        return result


import itertools


class Solution2(object):
    def createTargetArray(self, nums, index):
        """
        :type nums: List[int]
        :type index: List[int]
        :rtype: List[int]
        """
        result = []
        for i, x in itertools.izip(index, nums):
            result.insert(i, x)
        return result
